docs/src/user-docs/quick-start/clp-text.md

```
@@ -153,8 +276,49 @@ The numbered circles in [Figure 3](#figure-3) correspond to the following elemen
+
+```bash
+curl -X POST "http://localhost:3001/query/submit" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "query_string": "<query>",
+    "max_num_results": 1000,
+    "timestamp_begin": null,
+    "timestamp_end": null,
+    "case_sensitive": false
+  }'
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)